src/applications/vaos/project-cheetah/components/VAFacilityPage/index.jsx

```diff
@@ -100,7 +99,10 @@ function VAFacilityPage({
 
   const title = <h1 className="vads-u-font-size--h2">{pageTitle}</h1>;
 
-  if (hasDataFetchingError) {
+  if (
+    facilitiesStatus === FETCH_STATUS.failed ||
+    (clinicsStatus === FETCH_STATUS.failed && singleValidVALocation)
```

Comment:
This allows clinic errors to pass through to the modal, which allows users to change facilities and try again